esss_fix_format/cli.py

```diff
@@ -218,6 +218,10 @@ def get_files(cmd):
     result.update(get_files('git diff --name-only --diff-filter=ACM --staged'))
     result.update(get_files('git diff --name-only --diff-filter=ACM'))
     result.update(get_files('git ls-files -o --full-name --exclude-standard'))
+    # check_output returns bytes in Python 3
+    if sys.version_info[0] == 3:
+        result = [x.decode(sys.getfilesystemencoding()) for x in result]
```

Comment:
https://www.python.org/dev/peps/pep-0529/#background

This section of PEP-529 has some helpful docs related to this.